xendit_pos/views/point_of_sale_assets.xml

```diff
@@ -2,9 +2,9 @@
 <odoo>
     <template id="assets" inherit_id="point_of_sale.assets">
         <xpath expr="." position="inside">
-            <script type="text/javascript" src="/xendit/static/src/js/payment_xendit.js"></script>
-            <script type="text/javascript" src="/xendit/static/src/js/models.js"></script>
-            <script type="text/javascript" src="/xendit/static/src/js/xendit_qrcode_popup.js"></script>
+            <script type="text/javascript" src="/xendit_pos/static/src/js/payment_xendit_pos.js"></script>
+            <script type="text/javascript" src="/xendit_pos/static/src/js/models.js"></script>
+            <script type="text/javascript" src="/xendit_pos/static/src/js/xendit_qrcode_popup.js"></script>
```

Comment:
I'm thinking that the name of the root folder should be `xendit_odoo_pos` for clarity. If we use `xendit_pos` it sounds generic to any POS. wdyt?

Reply:
> Looks generally OK. some comments that can be considered on a separate PR

Thanks @rendegosling . I will update them on the next PR.